core/lib/resources/presence/presence_manager.js

```diff
@@ -51,15 +51,15 @@ PresenceManager.prototype.setup = function() {
   this.sentryListener = function (sentry) {   
     
     var socketIds = store.socketsForSentry(sentry);
```

Comment:
they're not `undefined`, `socketIds` is an empty array, and `[].pop() => undefined`.